whd/dispatcher.py

```diff
@@ -79,11 +76,6 @@ def concourse_clients(
     def dispatch_create_event(
         self,
         create_event,
-        delivery_id: str,
```

Comment:
no longer passing elasticsearch-client makes sense, but why also drop the other parameters? were those only used for reporting into ES (in which case, it is of course reasonable to drop)?

question also applies to subsequent similar code-changes

Reply:
yes, additional parameters have only been used for elasticsearch metrics